contrib/linux/install-icons.sh

```diff
@@ -25,14 +25,16 @@ usage() {
 
 install_icons() {
     cp -r icons $DEST_PATH
-    cp desktop/dosbox-staging.desktop $DEST_PATH/applications
+    cp desktop/org.dosbox-staging.dosbox-staging.desktop $DEST_PATH/applications/org.dosbox-staging.dosbox-staging.desktop
 
     ## Replace executable location in the .desktop file
-    sed -i 's\Exec=dosbox\Exec='$(realpath dosbox)'\g' $DEST_PATH/applications/dosbox-staging.desktop
+    sed -i 's\Exec=dosbox\Exec='$(realpath dosbox)'\g' $DEST_PATH/applications/org.dosbox-staging.dosbox-staging.desktop
 }
 
 remove_icons() {
     find $DEST_PATH/icons -name dosbox-staging* -delete
+    rm $DEST_PATH/applications/org.dosbox-staging.dosbox-staging.desktop
```

Comment:
Maybe extract `org.dosbox-staging.dosbox-staging.desktop` to a var?

Reply:
OK, changed.